.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,80 @@
+name: Release Docker hub
+on:
+  push:
+    tags:
+    - '*.*.*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  main:
+    env:
+      BASE_PROJECT: arm-cli
+      GRGIT_USER: GitHub
+      GRGIT_PASS: ${{ secrets.GITHUB_TOKEN }}
+      JFROG_ARGS: --url https://armory.jfrog.io/armory/ --user ${{secrets.ARTIFACTORY_USER}} --password ${{secrets.ARTIFACTORY_PASSWORD}}
+      GRADLE_ARGS: -Partifactory_user=${{secrets.ARTIFACTORY_USER}} -Partifactory_password=${{secrets.ARTIFACTORY_PASSWORD}} --info -x dockerClean -x dockerPrepare
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get the version
+        id:   get_version
+        run:  echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: Cache Build Dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: gradle-${{ hashFiles('**/*.gradle*') }}
+      
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Fetch full history
+        run: git fetch --prune --tags --unshallow
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
```

Comment:
Probably should make this `11`, `1.8` has been EOL a while

Reply:
Done!